### ISSUE TITLE
Is it possible to rebind what `e` key does in normal mode to do `yank` instead?

### ISSUE THREAD

**Issue by nikitavoloboev** [VSCodeVim/Vim#5321]
I want to rebind `e` key in normal mode to instead of going to end of word, to do `copy` or `yank` in vim terms.

Is it possible to do?

**Comment by dobrovolsky** [comment]
You can try add something like this to your `settings.json`
```json
"vim.normalModeKeyBindingsNonRecursive": [
    {
      "before": ["e"],
      "after": ["y", "i", "w"]
    }
]
```

**Comment by nikitavoloboev** [comment]
Both your suggestion and this:

```json
  "vim.normalModeKeyBindings": [
    {
      "before": ["e"],
      "after": ["y"]
    },
```

Didn't work for me. It still did what `e` does, go to end of word.

**Comment by berknam** [comment]
Can you give more information about your system, vscode and extension versions?

Both the remaps above should work. Are you trying to switch those two keys? Do you have some other remap for `y`?

**Comment by glm4610** [comment]
Maybe unrelated, but [this solution](https://github.com/Microsoft/vscode/issues/24043#issuecomment-292334048) fixed some problems I had after remapping some keys around:

> Workaround to switch VS Code to dispatch based on key code again. Add the following setting:
> `"keyboard.dispatch": "keyCode"` and restart VS Code

**Comment by J-Fields** [comment]
@nikitavoloboev Is this still an issue you're having?

**Comment by nikitavoloboev** [comment]
@J-Fields Yes, still an issue for me as @dobrovolsky solution didn't work for me.

**Comment by J-Fields** [comment]
@nikitavoloboev Can you please post your settings? What does `e` do after creating a remapping?
